uberstyle/README.md

```diff
@@ -0,0 +1,76 @@
+  - [Use go.uber.org/atomic](https://github.com/uber-go/guide/blob/master/style.md#use-gouberorgatomic)
+  - [Avoid Mutable Globals](https://github.com/uber-go/guide/blob/master/style.md#avoid-mutable-globals)
+  - [Avoid Embedding Types in Public Structs](https://github.com/uber-go/guide/blob/master/style.md#avoid-embedding-types-in-public-structs)
+  - [Avoid Using Built-In Names](https://github.com/uber-go/guide/blob/master/style.md#avoid-using-built-in-names)
+  - [x] [Avoid `init()`](https://github.com/uber-go/guide/blob/master/style.md#avoid-init) - Style applied through `gochecknoinits`.
+  - [x] [Exit in Main](https://github.com/uber-go/guide/blob/master/style.md#exit-in-main) - Style applied through `revive.deep-exit`.
+  - [x] [Use field tags in marshaled structs](https://github.com/uber-go/guide/blob/master/style.md#use-field-tags-in-marshaled-structs) - Style applied through `musttag`
+  - [Don't fire-and-forget goroutines](https://github.com/uber-go/guide/blob/master/style.md#dont-fire-and-forget-goroutines)
+    - [Wait for goroutines to exit](https://github.com/uber-go/guide/blob/master/style.md#wait-for-goroutines-to-exit)
+    - [No goroutines in `init()`](https://github.com/uber-go/guide/blob/master/style.md#no-goroutines-in-init)
+- [Performance](https://github.com/uber-go/guide/blob/master/style.md#performance)
+  - [x] [Prefer strconv over fmt](https://github.com/uber-go/guide/blob/master/style.md#prefer-strconv-over-fmt) - Style applied through `perfsprint`
+  - [Avoid repeated string-to-byte conversions](https://github.com/uber-go/guide/blob/master/style.md#avoid-repeated-string-to-byte-conversions)
+  - [Prefer Specifying Container Capacity](https://github.com/uber-go/guide/blob/master/style.md#prefer-specifying-container-capacity)
+- [Style](https://github.com/uber-go/guide/blob/master/style.md#style)
+  - [x] [Avoid overly long lines](https://github.com/uber-go/guide/blob/master/style.md#avoid-overly-long-lines) - Style applied through `lll`
+  - [Be Consistent](https://github.com/uber-go/guide/blob/master/style.md#be-consistent)
+  - [Group Similar Declarations](https://github.com/uber-go/guide/blob/master/style.md#group-similar-declarations)
+  - [Import Group Ordering](https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering)
+  - [Package Names](https://github.com/uber-go/guide/blob/master/style.md#package-names)
+  - [Function Names](https://github.com/uber-go/guide/blob/master/style.md#function-names)
+  - [Import Aliasing](https://github.com/uber-go/guide/blob/master/style.md#import-aliasing)
+  - [x] [Function Grouping and Ordering](https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering) - Style applied through linter `funcorder`.
+  - [Reduce Nesting](https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting)
+  - [Unnecessary Else](https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else)
```

Comment:
There is something for both in a least revive

revive.superflous-else
revive.if-return
revive.early-return
revive.unnecessary-stmt
revive.indent-error-flow


But please also check at go-critic